react/Arrow.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ * Arrow component. It's an overrided component of react-slick that controls 
+ * the slide trasition.
+ */
+class Arrow extends Component {
+  render() {
+    const { className, style, onClick, color } = this.props
+    return (
+      <div
+        className={className}
+        style={{ ...style, color: color, fontSize: '210%' }}
```

Comment:
Usar um fontSize que nao seja em porcentagem. Se possível usar uma classe to Tachyons pra isso.

Reply:
@klzns  N consegui usando o tachyons n, problema de precedência. Tava pensando em usar um 'rem' fixo e colocar no theme.css mesmo, você acha que tem problema?